firstpr/alejandra_pinto.md

```diff
@@ -0,0 +1,10 @@
+### Hi there 👋
+
+My name is Alejandra, I´m Developer in process...
+----
+
+- 🔭 I’m currently working on Projects startup
+- 🌱 I’m currently learning Dart, Flutter and Phyton
+- 👯 I’m looking to collaborate on Mobile Apps.
+- 🤔 I’m looking for help with Haskell
```

Comment:
Remaining a . in the end of this sentence.

Reply:
all the reason.